qutebrowser/browser/qutescheme.py

```diff
@@ -39,6 +40,7 @@
 from qutebrowser.utils import (version, utils, jinja, log, message, docutils,
                                objreg, urlutils)
 from qutebrowser.misc import objects
+import sip
```

Comment:
Since this is a third-party and not a qutebrowser import, it should be in the second block where `pkg_resources` is imported.